netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -726,6 +728,7 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
       // which causes the upgrade headers to be added
       DefaultHttpRequest upgradeTrigger =
           new DefaultHttpRequest(HttpVersion.HTTP_1_1, HttpMethod.GET, "/");
+      upgradeTrigger.headers().add(HttpHeaderNames.HOST, "");
```

Comment:
This should be /something/ right?

Reply:
Seems like it should be the authority. See how `TlsNegotiator`'s host+port is plumbed, partly in this file.

Reply:
authority was my guess too, but I don't know if it maps exactly to HTTP/1.1 Host.  Also, what happens if both are present? o_O